manifest.php

```diff
@@ -54,7 +55,8 @@
 		),
 		'php'	=> array(
             CreateItemDirectory::class,
-            RegisterCategoryService::class
+            RegisterCategoryService::class,
+            RegisterNpmPaths::class
```

Comment:
You can add a trailing comma on every line in php arrays, this makes the adding of line a one-line diff...

Reply:
Fixed